generative_data_prep/processors/article_tokenizer.py

```diff
@@ -133,8 +138,37 @@ def __call__(self, article: Optional[str]) -> List[TokenizedSequence]:
 
         tokenized_sequences = self.packer(tokenized_articles)
```

Comment:
I thought we decided on removing sequences before the packer as well to avoid the issue where we have EOS token without a completion

Reply:
Try to add this in as clean a way as possible. Maybe in the TokenizedArticle class or Article Tokenizer class, so it is never actually added to the list of tokenized articles

Reply:
Hm I'll probably keep it in the ArticleTokenizer class as it is and use the original implementation. Don't think I can make it a Tokenized<> class and make it remove itself from a list